src/main/java/io/openmanufacturing/ame/web/PackageResource.java

```diff
@@ -94,11 +93,11 @@ public ResponseEntity<List<String>> importAspectModelPackage( @RequestBody final
    /**
     * Method to export packaged aspect models.
     *
-    * @param zipFileName - the name of the zip file to be created.
     * @return the package as zip file as blob.
     */
-   @GetMapping( path = "/export-zip/{zipFileName}", produces = MediaTypeExtension.APPLICATION_ZIP_VALUE )
-   public ResponseEntity<byte[]> exportAspectModelPackage( @PathVariable final String zipFileName ) {
+   @GetMapping( path = "/export-zip", produces = MediaTypeExtension.APPLICATION_ZIP_VALUE )
```

Comment:
Instead of using zip I would recommend to rename it to export in the resource name. With that I would also rename the other resources accordingly e.g. validate. We should discuss that and create an issue to address the final result.